.github/pull_request_template.md

```diff
@@ -21,19 +35,15 @@ This PR CHANGES.
 
 ### Basics
 
-- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
+- [ ] PR assigned to `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa`
```

Comment:
Shouldn’t it be something like "PR assigned to `<developer>`" and "Add reviewers `input-output-hk/daedalus-dev` ...."? This way might cause confusion between PR assignee and PR reviewer

Reply:
Well, so this actually is another new thing for me - are there any rules about who (or how many people) should be assigned to the PR using the _Asignees_ field? I thought we only use _Reviewers_ and thought this is what the creator of the original checklist had in mind.